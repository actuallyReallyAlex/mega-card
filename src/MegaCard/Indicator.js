import React from 'react'
import { Box } from 'grommet'
import { colors, propTypes } from '../constants'

const Indicator = ({ color }) => {
  return (
    <Box>
      <Box
        height="8px"
        round="full"
        style={{ backgroundColor: colors[color] }}
        width="8px"
      />
    </Box>
  )
}

Indicator.propTypes = {
  color: propTypes.color
}

export default Indicator
