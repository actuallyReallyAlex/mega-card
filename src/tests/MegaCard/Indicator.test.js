import React from 'react'
import Indicator from '../../MegaCard/Indicator'
import { shallow } from 'enzyme'

const defaultProps = {}

describe('<Indicator />', () => {
  afterEach(() => {
    // defaultProps.handler.mockClear()
  })

  test('Should render the <Indicator /> component.', () => {
    const component = shallow(<Indicator {...defaultProps} />)
    expect(component).toMatchSnapshot()
  })
})
