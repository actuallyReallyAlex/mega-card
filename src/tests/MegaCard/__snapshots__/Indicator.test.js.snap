// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Indicator /> Should render the <Indicator /> component. 1`] = `ShallowWrapper {}`;
